lib/cn/spill.h

```diff
@@ -40,7 +55,30 @@ struct cn_compaction_work;
  */
 /* MTF_MOCK */
 merr_t
-cn_spill(struct cn_compaction_work *w);
+cn_subspill(
+    struct spillctx           *sctx,
+    struct subspill           *ss,
+    struct cn_tree_node       *node,
+    uint64_t                   node_dgen,
+    void                      *ekey,
+    uint                       eklen);
+
+
+/* MTF_MOCK */
+merr_t
+cn_spill_init(struct cn_compaction_work *w, struct spillctx **sctx_out);
+
+/* MTF_MOCK */
+void
+cn_spill_fini(struct spillctx *ctx);
+
+/* MTF_MOCK */
+struct subspill *
+cn_spill_get_nth_subspill(struct spillctx *sctx, uint n);
+
+/* MTF_MOCK */
+void
+cn_subspill_get_kvset_meta(struct subspill *ss, struct kvset_meta *km);
```

Comment:
For consistency, I would make struct subspill * be the first arg for every function that starts with "cn_subspill".